enterprise/backend/test/metabase_enterprise/public_settings_test.clj

```diff
@@ -13,16 +14,39 @@
                                      jwt-identity-provider-uri "example.com"
                                      jwt-shared-secret         "0123456789012345678901234567890123456789012345678901234567890123"
                                      enable-password-login     true]
```

Comment:
You're going to have to use `(premium-features-test/with-premium-features #{:sso-jwt} ...)` around this block now that https://github.com/metabase/metabase/pull/32472 is merged

Reply:
merged with upstream and that is wrapped now [28e5472](https://github.com/metabase/metabase/pull/32464/commits/28e5472fecb71492d32f0608de6b74910679c602)